lib/perl/Genome/VariantReporting/Command/Wrappers/GoldEvaluation.pm

```diff
@@ -86,5 +88,89 @@ sub is_single_bam {
     return (!defined($model->normal_sample));
 }
 
+sub get_common_translations {
```

Comment:
This is only used in the test - did you intend to use it at lines 39 and 48?  I actually don't see much point in it, since both those use cases already have the is_single_bam conditional